docs/reference/running/deployment/note_cloud_reset_password.rst

```diff
@@ -0,0 +1,7 @@
+.. note:: Gel Cloud: Reset the default password for the admin role
+
+    If you want to dump an existing Gel Cloud instance, you will need to change the automatically generated password for the standard admin role - ``edgedb`` or ``admin``. To do this, execute the following query in the instance:
```

Comment:
Does the password need to be changed to the one used in the dump?

Reply:
I guess it should also call out that the admin user must be the same as the dump too.